### DISCUSSION TITLE
Layout section should be applicable per-monitor

### DISCUSSION THREAD

**Discussion by morenathan** [YaLTeR/niri#354] [Ideas]
For multi-screen setups it would be desirable to have the ability to use the functions of the layout section on each screen if desired.

For example, I have a 1080p display to my left rotated 90 degrees. I use this for documentation, messaging, etc, but the problem is the initial window setup. This screen will never want a 50/50 split which is my preferred setup on the main screen which is 1440p and not rotated.

Thank you for a wonderful project! (yes, I'm still trying to learn rust; just busy)

**Comment by YaLTeR** [comment]
Hmm, that's a good point. I'll need to think what would be the best way of doing this.
